spec/defines/filter/client_auth_n_spec.rb

```diff
@@ -190,8 +190,9 @@
           'pass' => 'password',
           'uri'  => 'http://uri'
         },
-        :client_maps => [ 'testing', ],
-        :delegating  => 'true'
+        :client_maps         => [ 'testing', ],
+        :delegating          => 'true',
+        :delegating_quality  => '0.9'
```

Comment:
The whole point was that in the ruby/rspec work we cant end the last item in a dict or list with a comma, but we do it in the puppet dsl. was just making a note.